docs/source/configuration.rst

```diff
@@ -57,6 +57,76 @@ Cloud provider configuration
     [cloud-provider]
     gcp-zone = us-east4-b
 
+.. _elb_aws_region:
+
+``AWS Region``
+^^^^^^^^^^^^^^
+
+    Name of the AWS region to use. Recommended value: ``us-east-1``.
+
+    * Default: None
+    * Values: String, any region that supports Batch, see `AWS documentation for details <https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/>`_
+
+    Also supported via the environment variable: ``ELB_GCP_REGION``.
```

Comment:
This will be quite confusing. Maybe we should rename the environment variable to ELB_CLOUD_REGION.

Reply:
Noted, we'll address this in the next release.